chalicelib/checks/lifecycle_checks.py

```diff
@@ -28,6 +28,7 @@ def check_file_lifecycle_status(connection, **kwargs):
     check.summary = ""
     check.full_output = {}
     check.status = "PASS"
+    check.allow_action = True
```

Comment:
Solely as an FYI so you've seen it: I included some helpers awhile back to set some of these things automatically https://github.com/dbmi-bgm/foursight-cgap/blob/master/chalicelib/checks/helpers/utils.py#L10-L34